dynamo/plot/utils.py

```diff
@@ -580,7 +580,7 @@ def _matplotlib_points(
             ax.legend(
                 handles=legend_elements,
                 bbox_to_anchor=(1.04, 1),
-                loc=show_legend,
+                # loc=show_legend,
```

Comment:
why do you comment this?

Reply:
fixed by Xiaojie in the separate commits.